QModManager/API/SMLHelper/Logger.cs

```diff
@@ -0,0 +1,109 @@
+﻿namespace QModManager.API.SMLHelper
+{
+    using System;
+    using System.IO;
+    using System.Reflection;
+
+    internal enum LogLevel
+    {
+        Debug = 0,
+        Info = 1,
+        Warn = 2,
+        Error = 3,
+    }
+
+    internal static class Logger
```

Comment:
Use qmod logger?

Reply:
The SMLHelper logger and QMod logger should both be fairly similar.  
Yes. We should unify them.